docker.go

```diff
@@ -438,6 +438,18 @@ func (d *docker) stopContainer(ctx context.Context, id string) error {
 	return nil
 }
 
+func (d *docker) removeContainer(ctx context.Context, id string) error {
+	d.lock.Lock()
+	defer d.lock.Unlock()
+
+	err := d.client.ContainerRemove(ctx, id, types.ContainerRemoveOptions{})
```

Comment:
I think it would be better to use `Force: true` here, just in case.

Reply:
I added it here: https://github.com/orlangure/gnomock/pull/757/commits/a2b478c58f12c6efb0e3c2e9a18e4feb62bfe54a